python-sgd-regression/db_models.py

```diff
@@ -0,0 +1,25 @@
+# TODO: this module with ORM for database should go to io_helpers
+# QUESTION: does it make sense to use heavy ORM like SQLALchemy or should I simply use raw SQL through SqlAlchemy? (or
+# maybe different ORM?)
+
```

Comment:
Let's use SQLAlchemy for JobResult. Otherwise I'm happy with SqlAlchemy + raw SQL